test/cases/gradient.out.css

```diff
@@ -220,3 +220,37 @@ div {
   background-image: url("https://test.com/lol(test.png"), -o-radial-gradient(yellow, black, yellow);
   background-image: url("https://test.com/lol(test.png"), radial-gradient(yellow, black, yellow);
 }
+
+.unitless-zero {
+  background-image: -webkit-gradient(linear, left bottom, left top, from(green), to(blue));
+  background-image: -webkit-linear-gradient(bottom, green, blue);
+  background-image: -o-linear-gradient(bottom, green, blue);
+  background-image: linear-gradient(0, green, blue);
+  background: -webkit-repeating-linear-gradient(bottom, blue, red 33.3%);
+  background: -o-repeating-linear-gradient(bottom, blue, red 33.3%);
+  background: repeating-linear-gradient(0, blue, red 33.3%)
+}
+
+.zero-grad {
+  background: -webkit-gradient(linear, left bottom, left top, from(green), to(blue));
+  background: -webkit-linear-gradient(bottom, green, blue);
+  background: -o-linear-gradient(bottom, green, blue);
+  background: linear-gradient(0grad, green, blue);
+  background-image: -webkit-repeating-linear-gradient(bottom, blue, red 33.3%);
```

Comment:
I don't know why the test failed here, the single test(`test.only('changes angle in gradient'`) was passed.

Reply:
It was prefixes cleaning test which failed.

Seems like we forgot to think about cleaning repeating gradients when we added their support.

Here is my fix 542bd85

Reply:
Nice